### DISCUSSION TITLE
Is there auto cut silence in 0.2?

### DISCUSSION THREAD

**Discussion by ImmanuelJeswyn** [olive-editor/olive#1961] [Q&A]
I come across olive video editor when I was searching for auto silent remover tools. I found version 0.1 has this feature and still works great. But for my surprise 0.2 alpha dont have this feature. Am I missing it or will it be added eventually?

**Comment by algopenne** [comment]
Would love this feature as well!

**Comment by wojtek-tp** [comment]
+1

**Comment by alexmyczko** [comment]
Well there is external software for just that: https://tracker.debian.org/pkg/auto-editor
